### PR TITLE
Add force delete for other directories and add ico extension

### PR DESCRIPTION
**Pull request by Barraki**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ function deleteUnusedImages(options) {
         input: fs.createReadStream(String(chunk.path)),
         terminal: false
       }).on('line', function (line) {
-        var filename = (line.match(/((?:((?:[^\(\\\'\"\r\n\t\f\/\s\.])+)\.(?:(png|gif|jpe?g|pdf|xml|apng|svg|mng)\b)))/gmi) || []).pop();
+        var filename = (line.match(/((?:((?:[^\(\\\'\"\r\n\t\f\/\s\.])+)\.(?:(png|gif|jpe?g|pdf|xml|apng|svg|ico|mng)\b)))/gmi) || []).pop();
```
Comment (rickysullivan) [inline index.js:46]: Sure.

File: index.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ function deleteUnusedImages(options) {
 
     if (unusedImages.length) {
       if (options.delete) {
-        del(unusedImages).then(paths => {
+        del(unusedImages, {force: true}).then(paths => {
```
Comment (rickysullivan) [inline index.js:79]: Reluctant to default force delete to true. Is there a reason why you need to force delete images?

**Review by rickysullivan — COMMENTED** [review]
Update

File: index.js (hunk 2 above)
Comment (Barraki) [reply index.js:79]: It is impossible to delete images if the images are not in the directory with node_modules. In projects with monolith architecture

**Pull request closed without merging**